roles/installer/templates/networking/ingress.yaml.j2

```diff
@@ -27,6 +33,15 @@ spec:
                 name: '{{ ansible_operator_meta.name }}-service'
                 port:
                   number: 80
+{% if ingress_controller|lower == "contour" %}
+          - path: '{{ ingress_path }}/websocket'
```

Comment:
@lumarel hey, could you give us some context for why it is only the `/websocket` path here?

Could you also describe the use case for configuration like this?  I'm not very familiar with Contour and Envoy, thanks ahead of time!

Reply:
Ok, I re-read your description, I understand now, thanks.